### PR TITLE
MOSIP-13273 mosip regprc db changes

### PR DESCRIPTION
**Pull request by rambhatt1591**

### CODE REVIEW

File: db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql [hunk 1]
```diff
@@ -13,13 +13,89 @@
...
+
+ALTER TABLE regprc.reg_lost_uin_det DROP CONSTRAINT pk_rlostd;
+ALTER TABLE regprc.reg_lost_uin_det ALTER COLUMN workflow_instance_id SET NOT NULL;
+ALTER TABLE regprc.reg_lost_uin_det ADD CONSTRAINT pk_rlostd PRIMARY KEY (workflow_instance_id);
+
+ALTER TABLE regprc.reg_manual_verification DROP CONSTRAINT pk_rmnlver_id;
+ALTER TABLE regprc.reg_manual_verification ALTER COLUMN workflow_instance_id SET NOT NULL;
+ALTER TABLE regprc.reg_manual_verification ADD CONSTRAINT pk_rmnlver_id PRIMARY KEY (workflow_instance_id,matched_ref_id,matched_ref_type);
+
+ALTER TABLE regprc.registration DROP CONSTRAINT pk_reg_id;
```
Comment (MonobikashDas) [inline db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql:93]: currently "pk_reg_id" is used as foreign key in below tables. So the CONSTRAINT cant be dropped without altering below tables. Please remove CONSTRAINT from below tables and add workflow_instance_id as foreign key
constraint fk_idemogd_reg on table individual_demographic_dedup depends on index pk_reg_id
constraint fk_rmnlver_reg on table reg_manual_verification depends on index pk_reg_id
constraint fk_regbrf_reg on table reg_bio_ref depends on index pk_reg_id
constraint fk_rlostd_reg on table reg_lost_uin_det depends on index pk_reg_id

**Review by MonobikashDas — CHANGES_REQUESTED** [review]
currently "pk_reg_id" is used as foreign key in below tables. So the CONSTRAINT cant be dropped without altering below tables. Please remove CONSTRAINT from below tables and add workflow_instance_id as foreign key
constraint fk_idemogd_reg on table individual_demographic_dedup depends on index pk_reg_id
constraint fk_rmnlver_reg on table reg_manual_verification depends on index pk_reg_id
constraint fk_regbrf_reg on table reg_bio_ref depends on index pk_reg_id
constraint fk_rlostd_reg on table reg_lost_uin_det depends on index pk_reg_id

File: db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql (hunk 1 above)
Comment (rambhatt1591) [reply db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql:93]: This has been done.

File: db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql [hunk 2]
```diff
@@ -13,13 +13,112 @@
...
+-- Jul-2021		Ram Bhatt	   Added rows to transaction_type.csv
+-- Jul-2021		Ram Bhatt	   Multiple table changes on regprc db
 ----------------------------------------------------------------------------------------------------
 \c mosip_regprc sysadmin
 
+----------------------------------------------Multiple table level changes on regprc db-------------------------------------------------------
+
+ALTER TABLE regprc.individual_demographic_dedup DROP CONSTRAINT IF EXISTS fk_idemogd_reg CASCADE;
+ALTER TABLE regprc.reg_manual_verification DROP CONSTRAINT IF EXISTS fk_rmnlver_reg CASCADE;
+ALTER TABLE regprc.reg_bio_ref DROP CONSTRAINT IF EXISTS fk_regref_reg CASCADE;
```
Comment (MonobikashDas) [inline db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql:25]: drop CONSTRAINT fk_regbrf_reg for reg_bio_ref table

File: db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql [hunk 3]
```diff
@@ -13,13 +13,112 @@
...
+ALTER TABLE regprc.reg_lost_uin_det DROP CONSTRAINT IF EXISTS fk_rlostd_reg CASCADE;
+
```
Comment (MonobikashDas) [inline db_release_scripts/mosip_regprc/sql/1.2_regprc-scripts_release.sql:28]: additionally -
ALTER TABLE regprc.registration_transaction DROP CONSTRAINT IF EXISTS fk_regtrn_reg CASCADE

then add workflow_instance_id as foreign key for registration_transaction table

**Review by MonobikashDas — CHANGES_REQUESTED** [review]

**Review by MonobikashDas — APPROVED** [review]

**Pull request merged**